composer.json

```diff
@@ -14,7 +14,7 @@
     "require": {
         "php": ">=5.3.3",
         "guzzle/guzzle": "~3.7",
-        "symfony/console": "~2.5"
+        "symfony/console": "~2.5|~3.0|~4.0"
```

Comment:
It could be great to upgrade the whole stack, I mean support `"php": "^7.1"` and `"symfony/console": "^3.0 || ^4.0"`, WDYT?

Reply:
@mremi  I need PHP version 5.6. Could you release `v1.0.1` without update PHP version? And I would create new pull request with all stack update for another version `2.0.0`?

Reply:
Ok for me, can you please fix the CI failure by removing support for PHP < 5.6? Just remove the lines in .travis.yml and update the composer.json with `^5.6`.

Reply:
Done. I guess now version should be `2.0.0` and version for PHP ^7.1 should be `3.0.0` :smile:

Reply:
You could require `"php": "^5.6 || ^7.1"` and remove 7.0 in travis, not supported anymore soon

Reply:
@mremi Done.